src/main.rs

```diff
@@ -297,6 +307,10 @@ fn match_vulnerability(vuln: &Vulnerability, opts: &AuditOpts) -> bool {
         }
     }
 
+    if opts.ignore.contains(&vuln.advisory.id.as_str().to_owned()) {
+        return false;
+    }
```

Comment:
Hmm, branching on a boolean to determine a boolean return value.  It feels both unnecessary, but also like removing it wouldn't particularly improve code clarity. What do you think?

Reply:
I agree there is a bit of tension. I chose to stick with the local code style since it does have the benefit of making each filter check added read as independently affecting the result:
```
[filter1]
[filter2]
[filter3]
return true
```

Switching would be more concise but I think less clear. Now the `ignore` check works a bit different from all the rest in the priviledged final position.
```
[filter1]
[filter2]
return ![filter3]
```

Reply:
Seems ok